### ISSUE TITLE
OCaml Planet not updating?

### ISSUE THREAD

**Issue by talex5** [ocaml/v2.ocaml.org#1620]
https://ocaml.org/community/planet/ shows no updates since Oct 21, 2021, which seems unlikely. Has it stopped?
(my own blog is listed under Syndications and was updated on  2021-10-30, for example)

**Comment by patricoferris** [comment]
We haven't merged anything to `master` since 27th October, so the site hadn't been rebuilt since then :|

I merged a simple PR and looks like your post is now live https://ocaml.org/community/planet/ :))

**Comment by talex5** [comment]
Ah, I didn't realise it was manual!

**Comment by patricoferris** [comment]
Yeah, it's not ideal, perhaps we can do something in the deployer to rebuild every week or so regardless of commits to `master` ?
